src/app/components/blog/blog.component.ts

```diff
@@ -21,8 +19,8 @@ export class BlogComponent implements OnInit {
       this.getCards().subscribe((results) => {
         this.blogPost = results['items'];
         this.totalLength = results['items'].length;
-        console.log(this.blogPost);
-        console.log(this.totalLength);
+        //console.log(this.blogPost);
+        //console.log(this.totalLength);
```

Comment:
remove these comments

Reply:
I thought to keep this for future need. Actually added by @Bawanthathilan

Reply:
can someone remove it